File: 프로그래머스/2/12924. 숫자의 표현/숫자의 표현.js
function solution(n) {
    var answer = 1;
    for(let i=1; i< Math.ceil(n/2)+1; i++){
        let num = 0;
        num += i;
        for(let j=i+1; j<Math.ceil(n/2)+1; j++){
            num += j;
            if(num < n) continue;
            else if(num > n) break;
            else {
                answer++;
                break;
            }
        }
    }
    
    return answer;
}

File: 프로그래머스/2/12924. 숫자의 표현/숫자의 표현.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        for(int i=1; i<= n; i+=1){
            int count = 0;
            for(int j=i; j<= n; j+=1){
                count += j;
                if(count == n){
                    answer +=1;
                    break;
                }else if(count > n){
                    break;
                }
            }
        }
        return answer;
    }
}